server/text_generation_server/models/causal_lm.py

```diff
@@ -630,6 +642,52 @@ def __init__(
             self.hb_profer_started = False
         self.step = 0
 
+    def setup_env(self):
+        import sys
+        assert "habana_frameworks" not in sys.modules
+
+        self.is_quantization_enabled = os.getenv("QUANT_CONFIG", "") != ""
+        if self.is_quantization_enabled:
+            os.environ.setdefault("ENABLE_EXPERIMENTAL_FLAGS", "true")
```

Comment:
Is it needed? Enabling experimental flags is restricted.

Reply:
yes, it's needed